### PR TITLE
Update the line length to 120

### PR DESCRIPTION
**Pull request by kobihk**
Setting the line length to 120 characters is a common and widely accepted practice, especially in the Python community. The recommendation comes from the PEP 8 style guide, which suggests limiting all lines to a maximum of 79 characters for code and 72 characters for docstrings. However, these recommendations were made when many developers were using terminals with a standard width of 80 characters.

In modern development environments, where wider screens are more common, the PEP 8 recommendation has been relaxed, and it's common to see a maximum line length of 120 characters.

### CODE REVIEW

File: .github/workflows/code_quality.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
           for file in ${{ steps.changed-files.outputs.all_changed_files }}; do
             echo "$file was changed"
             if [[ $file == *.py ]]; then
-               black $file --check --diff
+               black $file --check --diff --line-length 120
```
Comment (jiridanek) [inline .github/workflows/code_quality.yaml:25]: consider configuring this in pyproject.toml, https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#configuration-via-a-file

c.f.

https://github.com/red-hat-data-services/ods-ci/blob/7df1a217ca7668a3a4983f47f27995439864fc27/pyproject.toml#L48

https://github.com/red-hat-data-services/ods-ci/blob/7df1a217ca7668a3a4983f47f27995439864fc27/pyproject.toml#L52

**Review by jiridanek — CHANGES_REQUESTED** [review]

File: pyproject.toml [hunk 2]
```diff
@@ -45,11 +45,11 @@ build-backend = "poetry.core.masonry.api"
 [tool.isort]
```
Comment (jiridanek) [inline pyproject.toml:45]:
```suggestion
[tool.black]
line-length = 120
target-version = ['py311']

[tool.isort]
```

**Review by jiridanek — CHANGES_REQUESTED** [review]

File: pyproject.toml [hunk 3]
```diff
@@ -42,14 +42,18 @@ ruff = "0.1.11"
...
+[tool.black]
+line-length = 120
+target-version = ['py311']
+
 [tool.isort]
-# https://pycqa.github.io/isort/docs/configuration/black_compatibility.html
+# https://pycqa.github.io/isort/docs/configurationtoml file/black_compatibility.html
 profile = "black"
-line_length = 88 # align with black's default
+line-length = 120
```
Comment (jiridanek) [inline pyproject.toml:52]:
```suggestion
line_length = 120
```

typo

File: pyproject.toml [hunk 4]
```diff
@@ -42,14 +42,18 @@ ruff = "0.1.11"
 requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
 
+[tool.black]
+line-length = 120
+target-version = ['py311']
+
 [tool.isort]
-# https://pycqa.github.io/isort/docs/configuration/black_compatibility.html
+# https://pycqa.github.io/isort/docs/configurationtoml file/black_compatibility.html
```
Comment (jiridanek) [inline pyproject.toml:50]: ???

Comment (kobihk) [reply pyproject.toml]: good catch :)

**Review by jiridanek — APPROVED** [review]

**Review by jstourac — APPROVED** [review]
Love that, thank you! :tada:

**Pull request merged**